pytrip/res/concave_tool_v2.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def to_ind(mm, pix_size, x_off, y_off, z_off, s_thick):
+    return [
+        int((mm[0] - x_off) / pix_size),
+        int((mm[1] - y_off) / pix_size),
+        int((mm[2] - z_off) / s_thick)
+    ]
+
+
+def create_contour(intersections_list_mm, plane, x_size, y_size, z_size, pixel_size, x_offset, y_offset, z_offset,
+                   slice_thickness):
+    bitmap = None
+    if plane == 'Sagittal':

```

Comment:
Maybe use an enum.

Reply:
there is an issue for that https://github.com/pytrip/pytripgui/issues/440